cmd/glasskube/cmd/bootstrap.go

```diff
@@ -83,13 +84,24 @@ var bootstrapCmd = &cobra.Command{
 			fmt.Fprintf(os.Stderr, "\nAn error occurred during bootstrap:\n%v\n", err)
 			cliutils.ExitWithError()
 		}
-		if err := printBootsrap(
-			manifests,
-			bootstrapCmdOptions.Output,
-		); err != nil {
-			fmt.Fprintf(os.Stderr, "\nAn error occurred in printing : %v\n", err)
-			cliutils.ExitWithError()
+		if bootstrapCmdOptions.dryrun {
```

Comment:
The code if `dryrun` is true, and the code in the `else` is the same – or am I missing something? If not, please remove the duplicate branch, and on that occasion, please fix the typo `printBootsrap` -> `printBootstrap`. Thanks!

Reply:
The `printBootstrap` function should be called in every case, sorry if that was unclear. It doesn't matter if `bootstrapCmdOptions.dryrun` is `true` or `false`, `printBootstrap` should be called anyway, just like it was before.